code/modules/gear_presets/uscm_ship.dm

```diff
@@ -128,18 +128,17 @@
 	minimap_background = MINIMAP_ICON_BACKGROUND_CIVILIAN
 
 /datum/equipment_preset/uscm_ship/reporter/load_gear(mob/living/carbon/human/new_human)
-	var/back_item = /obj/item/storage/backpack/satchel
 
 	new_human.equip_to_slot_or_del(new /obj/item/device/radio/headset/almayer/reporter(new_human), WEAR_L_EAR)
 	new_human.equip_to_slot_or_del(new /obj/item/clothing/under/marine/reporter(new_human), WEAR_BODY)
 	new_human.equip_to_slot_or_del(new /obj/item/clothing/suit/storage/jacket/marine/reporter(new_human), WEAR_JACKET)
 	new_human.equip_to_slot_or_del(new /obj/item/clothing/shoes/laceup(new_human), WEAR_FEET)
-	new_human.equip_to_slot_or_del(new back_item(new_human), WEAR_BACK)
-	new_human.equip_to_slot_or_del(new /obj/item/device/camera(new_human), WEAR_L_HAND)
-	new_human.equip_to_slot_or_del(new /obj/item/device/camera_film(new_human), WEAR_IN_BACK)
-	new_human.equip_to_slot_or_del(new /obj/item/device/binoculars(new_human), WEAR_IN_BACK)
-	new_human.equip_to_slot_or_del(new /obj/item/device/taperecorder(new_human), WEAR_IN_BACK)
-	new_human.equip_to_slot_or_del(new /obj/item/notepad(new_human), WEAR_IN_BACK)
+	new_human.equip_to_slot_or_del(new /obj/item/device/camera/broadcasting(new_human), WEAR_R_HAND)
+	new_human.equip_to_slot_or_del(new /obj/item/storage/pouch/general/large(new_human), WEAR_L_STORE)
+	new_human.equip_to_slot_or_del(new /obj/item/device/camera(new_human), WEAR_WAIST)
+	new_human.equip_to_slot_or_del(new /obj/item/device/fax_backpack(new_human), WEAR_BACK)
```

Comment:
It is my intention and yeah I really couldn't think of any other ways around this. I think its kinda silly in a good way though. Like Apone with his cigar but its just some poor underpaid reporter with way too much bulky and unwieldy gear

Reply:
You gave them a vendor, why not just put them in there?

Reply:
This would be giving the items to military CC too (I dont think there is a way to item lock the items from the civvie v military job title thing UNLESS I make a new vendor which their room is already packed tight)

Military CC already gets their gun and other gear; the fax combined with the broadcasting camera isn't really built for them or the RP they have

Reply:
A means to do this would be for the init_mob for the civilian to have `new_character.marine_buyable_categories = MARINE_CAN_BUY_ALL | CIVILIAN_CAN_BUY`

Then in __DEFINES/vendors add `#define CIVILIAN_CAN_BUY "civilian"` but not add this entry to `MARINE_CAN_BUY_ALL`

Then use CIVILIAN_CAN_BUY for a civilian only purchase in the vendor. It'll either need to be a pack, or you add multiple defines.

But there's likely other ways too.